packages/samples/headless-ssr/src/app/generic/components/facet.tsx

```diff
@@ -0,0 +1,40 @@
+import {FacetState, Facet as FacetController} from '@coveo/headless';
+import {useEffect, useState, FunctionComponent} from 'react';
+import FacetCommon from '../../common/facet';
+
+interface FacetProps {
+  title: string;
+  ssrState: FacetState;
+  controller?: FacetController;
+}
+
+export const Facet: FunctionComponent<FacetProps> = ({
+  title,
+  ssrState,
```

Comment:
Can we rename these to use the new terminology?

Reply:
good catch, I had run a build after updating from `master` and forgot about it when working on something else. the build failed due to some stuff like this